Sources/TokamakDOM/Views/Divider.swift

```diff
@@ -0,0 +1,29 @@
+// Copyright 2020 Tokamak contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import TokamakCore
+
+public typealias Divider = TokamakCore.Divider
+
+extension Divider: AnyHTML {
+  var innerHTML: String? { nil }
+  var tag: String { "div" }
```

Comment:
**suggestion:** Wouldn't `hr` work better here, at least from the accessibility perspective it should convey the same semantic information as the `Divider` view, I don't know how well it matches the style though.

Reply:
I did use hr originally but changed the styling since hr has borders on all sides with different colors which is a bit weird. But I can switch it back and keep these styles. Should work the same.

Reply:
Yeah, I think we should stick to semantic elements if possible, for accessibility at least.